src/tests/client.js

```diff
@@ -0,0 +1,10 @@
+import ApolloClient from 'apollo-client'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { SchemaLink } from 'apollo-link-schema'
+
+import schema from '@api/schema'
```

Comment:
Do you always split imports like this? If you do, we could add a linting rule for that :D